### DISCUSSION TITLE
Modify 'mingw-w64-ucrt-x86_64-mlt' to include '.lib' files for seamless integration with Visual Studio.

### DISCUSSION THREAD

**Discussion by codewolf-in** [msys2/MINGW-packages#17714] [General]
I was wondering if it would be possible to modify the package 'mingw-w64-ucrt-x86_64-mlt' to include '.lib' files for potential use with Visual Studio. This could greatly enhance the integration between the package and Visual Studio, allowing developers to leverage its functionalities seamlessly within their projects.
I have attempted to generate a '.lib' file from a '.dll' using 'gendef' and 'lib.exe'. However, this approach results in an error indicating unresolved dependencies.

**Comment by Biswa96** [comment]
There is no plan to be compatible with Visual Studio. Also it is not possible due to various standard libraries. vcpkg may be helpful.

**Reply by codewolf-in** [reply]
Thank you for quick response.
